contributions/reporting.py

```diff
@@ -0,0 +1,80 @@
+"""
+===============================
+Functions for Report generation
+===============================
+File history and credits:
+G. Happle   ... 13.05.2016
+
+"""
+
+import pandas as pd
+import datetime
+import xlwt
+
+
+def full_report_to_xls(name, area_f, area_ef, sys_e_heating, sys_e_cooling, occupants, am, atot, awall_all,
+                       cm, ll, lw, retrofit, sh_typ, year, footprint, nf_ag, nfp, lcww_dis, lsww_dis, lv, lvww_dis,
+                       tcs_re_0, tcs_sup_0, ths_re_0, ths_sup_0, tww_re_0, tww_sup_0, y, fforma, ta_hs_set, ta_cs_set,
+                       people, ve_schedule, q_int, eal_nove, eprof, edata, qcdataf, qcrefirf, vww, vw, qcdata, qcrefri,
+                       qv_req, hve, htr_is, htr_ms, htr_w, htr_em, htr_1, htr_2, htr_3, i_sol, i_int_sen, w_int, i_ia,
+                       i_m, i_st, uncomfort, ta, tm, qhs_sen, qcs_sen, qhs_lat, qcs_lat, qhs_em_ls, qcs_em_ls, qhc_sen,
+                       ma_sup_hs, ta_sup_hs, ta_re_hs, ma_sup_cs, ta_sup_cs, ta_re_cs, w_sup, w_re, ehs_lat_aux,
+                       qhs_sen_incl_em_ls, qcs_sen_incl_em_ls, t5, tww_re, top, im_tot, thset_corr, tcset_corr, tcs_re,
+                       tcs_sup, ths_re, ths_sup, mcpcs, mcphs, mww, qww, qww_ls_st, qwwf, tww_st, vw_large, vww_large,
+                       mcpww, eaux_cs, eaux_fw, eaux_ve, eaux_ww, eauxf, occupancy, waterconsumption, location_final):
+
+    """ this function is to write a full report to an *.xls file containing all intermediate and final results of a
+     single building thermal loads calculation"""
+
+    # TODO: get names of used functions from glovalbars for report
+    # TODO: write units to column names
+    # TODO: split outputs into more work sheets, e.g. one for water, electricity, etc
+
+    # get date and time
+    now = datetime.datetime.now()
+    string_now = ("%s-%s-%s-%s-%s-%s" % (now.year, now.month, now.day, now.hour, now.minute, now.second))
```

Comment:
What is the pythonic way? Default formats of `datetime` strings include `.` or `:`, which can not be part of file names.